plugins/woocommerce/tests/legacy/unit-tests/payment-gateways/payment-gateways.php

```diff
@@ -42,7 +44,7 @@ public function test_wc_set_current_gateway_from_session() {
 
 		$gateways = WC()->payment_gateways()->get_available_payment_gateways();
 		$gateways['bacs']->chosen = false;
-		WC()->session->set( 'chosen_payment_method', 'bacs' );
+		WC()->session->set( 'chosen_payment_method', PaymentMethods::BACS );
```

Comment:
Just curious, why don't we use the new constant on the previous line? That is, like this:

```php
$gateways[PaymentMethods::BACS]->chosen = false;
```

(I'm only flagging this one, but I think there are a few areas like this.)